application/github/v0/config/tasks.json

```diff
@@ -0,0 +1,1607 @@
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 11,
+          "type": "string"
+        },
+        "direction": {
+          "default": "desc",
+          "title": "Direction",
+          "description": "Direction of the sort, including asc or desc. Default is asc",
+          "enum": [
+            "asc",
+            "desc"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 12,
+          "type": "string"
+        },
+        "since": {
+          "default": "2021-01-01T00:00:00Z",
+          "title": "Since",
+          "description": "Only comments updated at or after this time are returned. Default is 2021-01-01T00:00:00Z",
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 13,
+          "type": "string"
+        }
+      },
+      "required": [
+        "owner",
+        "repository"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Comments in the PR",
+      "instillUIOrder": 0,
+      "properties": {
+        "comments": {
+          "description": "An array of comments",
+          "instillUIOrder": 0,
+          "instillFormat": "object",
+          "title": "Comments",
+          "type": "array",
+          "items": {
+            "$ref": "#/$defs/review_comments",
+            "required": [],
+            "instillUIOrder": 1,
+            "description": "Comments in the PR"
+          }
+        }
+      },
+      "required": [
+        "comments"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_CREATE_REVIEW_COMMENT": {
+    "instillShortDescription": "Create a review comment in pull request.",
+    "description": "Create a review comment in a pull request. The comment can be a general comment or a review comment. The comment can be on a specific line or on the PR as a whole.",
+    "input": {
+      "description": "Please input the repository name and owner, and the PR number",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "owner",
+        "repository",
+        "pr_number"
+      ],
+      "properties": {
+        "$ref": "#/$defs/repository_info",
+        "pr_number": {
+          "title": "PR Number",
+          "description": "Number of the PR",
+          "instillFormat": "integer",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 3,
+          "type": "integer"
+        },
+        "comment": {
```

Comment:
Memo
Not sure the design here.
Why do the users need to insert commit_id, path, ... so many params?

Will come back to check it after reading more code as well.

Reply:
Because we need to specify what commit, file, and the range of lines to create a comment.